### ISSUE TITLE
Empty environment variables are not supported

### ISSUE THREAD

**Issue by glasserc** [jaredramirez/elm-constants#2]
To reproduce:

`.env`:

```
BASE_PATH=
```

`elm-constants.json`:

```
{
  "path": "./src/",
  "moduleName": "Env",
  "values": [
    "BASE_PATH"
  ]
}
```

Run `elm-constants`.

Expected behavior:

`Env.elm` produced with `basePath = ""`.

Actual behavior:

`I couldn't find any of the environment variable you specified, so I have nothing to generate!`

`BASE_PATH=""` of course does not change anything because it produces the same shell variable with the same shell value.

I guess the issue is here: https://github.com/jaredramirez/elm-constants/blob/bf6eefe3472a524c30cc2dc234a538adcca8906e/index.js#L139-L147

There's a confusion here between falsy environment variables and those which are missing.

I think there's also a larger issue here which is that we shouldn't try to be "safe" against missing environment variables. Elm code cannot detect undefined variables, so if a variable is truly missing, we should probably error out immediately rather than let later compilation steps fail when the variable is not there.

**Comment by jaredramirez** [comment]
Thanks for catching this an opening an issue.

I definitely agree that we should error immediately rather than let the compiler catch the issue later. I've fixed the falsy issue and added the error out immediate functionality. It's published as 0.0.3 on NPM now!

**Comment by glasserc** [comment]
Wow, that's fantastic. Thank you very much for your prompt response and thank you very much for your project! It's already made my life better, even before the bug was fixed :)
